tldextract/suffix_list.py

```diff
@@ -65,16 +77,20 @@ def get_suffix_lists(cache, urls, cache_fetch_timeout, fallback_to_snapshot):
     )
 
 
-def _get_suffix_lists(cache, urls, cache_fetch_timeout, fallback_to_snapshot):
+def _get_suffix_lists(
+    cache: DiskCache,
+    urls: Sequence[str],
+    cache_fetch_timeout: Union[float, int, None],
+    fallback_to_snapshot: bool,
+) -> Tuple[List[str], List[str]]:
     """Fetch, parse, and cache the suffix lists"""
 
     try:
         text = find_first_response(cache, urls, cache_fetch_timeout=cache_fetch_timeout)
     except SuffixListNotFound as exc:
         if fallback_to_snapshot:
-            text = pkgutil.get_data("tldextract", ".tld_set_snapshot")
-            if not isinstance(text, str):
```

Comment:
According to the type of `pkgutil.get_data`, this was always `bytes`, never `str`. 

(Technically `Optional[bytes]`. But we the package maintainers guarantee the inclusion of the file. So force unwrap it, below.)